app/models/order_detail.rb

```diff
@@ -250,7 +250,7 @@ def can_be_viewed_by?(user)
 
   # Supports single type or an array of product types
   scope :for_product_type, ->(product_type) { joins(:product).where(products: { type: product_type }) }
-  scope :untimed_orders, -> { for_product_type(["Item", "Service"]) }
+  scope :item_and_service_orders, -> { for_product_type(["Item", "Service", "TimedService"]) }
```

Comment:
It has to do with where the orders end up in the UI. Do they go under "My Orders" or "My Reservations" for the user who purchased them? Do they go under "Orders", "Reservations", or "Occupancies" on the admin side? Essentially, Reservations and Occupancies are treated somewhat specially, and then items and services are lumped together as the rest. `untimed_orders` sort of called this out better, but now that we have timed services that need to be in "the rest", it made at name not quite correct. This grouping is basically every product type that isn't special.